extensions/amp-analytics/0.1/test/test-variables.js

```diff
@@ -244,6 +244,14 @@ describes.fakeWin('amp-analytics.VariableService', {amp: true}, env => {
 
     it('if works', () => check('$IF(hey, truthy, falsey)', 'truthy'));
 
+    it('equals works (truth-y test)', () => {
```

Comment:
Can we also test 
```
$IF($EQUALS(${variable}, A), 1, $IF($EQUALS(${variable}, B), 2, 3));
```
I am a bit worried, that `$IF('false')` still resolve to true. Just to make sure.